drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -149,6 +149,97 @@ local child_device_profile_overrides = {
 
 local detect_matter_thing
 
+local CUMULATIVE_REPORTS_NOT_SUPPORTED = "__cumulative_reports_not_supported"
+local FIRST_EXPORT_REPORT_TIMESTAMP = "__first_export_report_timestamp"
+local EXPORT_POLL_TIMER_IS_SET = "__export_poll_timer_is_set"
+local EXPORT_REPORT_TIMEOUT = "__export_report_timeout"
+local TOTAL_EXPORTED_ENERGY = "__total_exported_energy"
+local LAST_EXPORTED_REPORT_TIMESTAMP = "__last_exported_report_timestamp"
+local RECURRING_EXPORT_REPORT_POLL_TIMER = "__recurring_export_report_poll_timer"
+local MINIMUM_ST_ENERGY_REPORT_INTERVAL = (15 * 60) -- 15 minutes, reported in seconds
+local SUBSCIPTION_REPORT_OCCURRED = "__subscription_report_occurred"
+
+local embedded_cluster_utils = require "embedded-cluster-utils"
+
+-- Include driver-side definitions when lua libs api version is < 11
+local version = require "version"
+if version.api < 11 then
+  clusters.ElectricalEnergyMeasurement = require "ElectricalEnergyMeasurement"
+  clusters.ElectricalPowerMeasurement = require "ElectricalPowerMeasurement"
+  clusters.ValveConfigurationAndControl = require "ValveConfigurationAndControl"
+end
+
+-- Return an ISO-8061 timestamp in UTC
+local function iso8061Timestamp(time)
+  return os.date("!%Y-%m-%dT%H:%M:%SZ", time)
+end
+
+local function delete_export_poll_schedule(device)
+  local export_poll_timer = device:get_field(RECURRING_EXPORT_REPORT_POLL_TIMER)
+  if export_poll_timer then
+    device.thread:cancel_timer(export_poll_timer)
+    device:set_field(RECURRING_EXPORT_REPORT_POLL_TIMER, nil)
+    device:set_field(EXPORT_POLL_TIMER_IS_SET, nil)
```

Comment:
Doesn't this value just follow the same logic as `RECURRING_EXPORT_REPORT_POLL_TIMER`? So could be just remove `EXPORT_POLL_TIMER_IS_SET` and just check that `RECURRING_EXPORT_REPORT_POLL_TIMER` is not nil where needed instead of having this separate flag?

Reply:
It's not exactly the same, since `EXPORT_POLL_TIMER_IS_SET` needs to gate some actions whether or not the poll timer is set. 

There are some lines that look like this:
>  if device:supports_capability(capabilities.powerConsumptionReport) then
>       create_poll_report_schedule(device)
>     end
>     device:set_field(EXPORT_POLL_TIMER_IS_SET, true)

and there are more places where the poll report may not be made, but we don't want to try this logic more than the once. Therefore, we need `EXPORT_POLL_TIMER_IS_SET` to stop this from being tried again.

Reply:
It could be argued that we should re-name the variable to `EXPORT_POLL_TIMER_SET_IS_TRIED` or something like that...

Reply:
I will change it to `EXPORT_POLL_TIMER_SETTING_ATTEMPTED`